Application/src/Main.java

```diff
@@ -30,12 +30,19 @@ public void start(Stage primaryStage) {
             public void handle(ActionEvent event) {
                 Collection<String> result = Import.file(primaryStage);
                 if (result == null) return;
-				// Check if the JSON file contains a supported diagram type.
-                System.out.println(DiagramCheck.ContainsDiagram(result));
-                // TODO remove print line and parse result (user story 5)
-                System.out.println(result);
-            }
-        });
+
+                // Parse the element if it contains a supported diagram
+                switch(DiagramCheck.ContainsDiagram(result)) {
+                    case "sequence_diagram" :
+                        for (String element : result ) {
+                            Parser.parseSequenceDiagram(element);
+                        }
+                        break;
+                    case "class_diagram" : //TODO
```

Comment:
2 cases that we dont need at the moment and no case for when the diagram is invalid